### PR TITLE
cc15 tree setup

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+change string to array, iterate through array.
+push open brackets into a stack, when closing bracket found, compare to top of stack.
+
+O(n) time & space
+
+## Solution
+
+- [Multi Bracket Validation](multi-bracket-validation.js) 
+
+![Whiteboard](assets/codechallenge13whiteboard.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree/README.md:17]: Looks like you are off to a start, I would suggest removing that multi bracket validation link out of here.
When you have time to come back and add code with tests, I will regrade for you!
